src/components/animation/animation-player.tsx

```diff
@@ -1,8 +1,17 @@
 import dynamic from "next/dynamic";
+import { Stack, Typography } from "@mui/material";
 
+const Loader = () => {
+  return (
+    <Stack>
+      <Typography variant={"h3"}>Loading...</Typography>
```

Comment:
Why not use the circular progress or the loading page component that we already have?

Reply:
Oops this was just a test, removed